src/main/java/api/ClassifyController.java

```diff
@@ -31,6 +31,18 @@
     @Value("${alto.data.base_dir:/usr/local/alto-boot}")
     String dataDirectory;
 
+    @Value("${alto.data.words_per_topic:20}")
+    int wordsNumPerTopic;
+
+    @Value("${alto.data.top_docs_per_topic:20}")
+    int docsNumPerTopic;
+
+    @Value("${alto.data.num_topics:5}")
+    int numTopics;
+
+    @Value("${alto.data.num_top_docs:100}")
```

Comment:
parameterized the max number of docs per topic to annotate and include in active learning predictions